<launch>
    <!-- Fake measurement generator -->
    <node pkg="cnnmot_ros" type="measurement_generator.py" name="fake_point_target_generator" output="screen">
        <param name="x_min" type="double" value="-5." />
        <param name="x_max" type="double" value="5." />
        <param name="y_min" type="double" value="-5." />
        <param name="y_max" type="double" value="5." />
        <param name="targets_max_nb" type="int" value="3" />
        <param name="measurement_noise" type="double" value="0.1" />
        <param name="target_velocity" type="double" value="1." />
        <param name="p_new_target" type="double" value="0.05" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find cnnmot_ros)/launch/config.rviz"/>
</launch>
